module/Dashboard/config/autoload/dao/NewRelicDao.config.php

```diff
@@ -31,10 +37,14 @@
                 'https://api.newrelic.com/v2/servers/:serverId:/metrics/data.json'
                 . '?names[]=System/Memory/Used/bytes&values[]=average_response_time'
                 . '&values[]=average_exclusive_time&from=:beginDateTime:&end=:endDateTime:&summarize=true',
-            'fetchMemoryAvgForNumberWidget' =>
+            NewRelicAddresses::AVG_HOST_MEMORY =>
```

Comment:
Previously, if you had the same data source (endpoint) but different widgets, you would have to duplicate addresses. I added some abstraction in NewRelicDao so that the dups are not needed.